src/herbivoresAndCarnivores.js

```diff
@@ -2,16 +2,46 @@
 
 class Animal {
   // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
```

Comment:
you don't have to put `health` as a constructor parameter, every instance of `Animal` have 100 of `health` by default
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
  }
```